docs/deployed-contracts.md

```diff
@@ -97,3 +97,12 @@
 - Curve Liquidity Farming:
 
   - Pool Contract: [`0x12edd9e2073E480cc546e1E0aD7F1c9D60c0cA1E`](https://goerli.etherscan.io/address/0x12edd9e2073E480cc546e1E0aD7F1c9D60c0cA1E)
+
+## Protocol Mocks
+
+### Rinkeby
+
+- Lido: [`0xF4242f9d78DB7218Ad72Ee3aE14469DBDE8731eD`](https://rinkeby.etherscan.io/address/0xF4242f9d78DB7218Ad72Ee3aE14469DBDE8731eD)
```

Comment:
add `StETH`, please

Reply:
I've updated the naming for this contract and others, to make it more consistent with the names of contracts in other sections.